### PR TITLE
Update properties of Search-AzGraph in regard to the new GA API version

### PR DESCRIPTION
**Pull request by fenwickt**
## Description

Added changes to ResourceGraph cmdlet Search-AzGraph to reflect recent GA version. 
- ManagementGroup param was added so that users can pass up to 10 management groups as a query scope
- AllowPartialScope param was added to support results for partial scope set if user query hits max number of scopes supported by the server
- Include param got removed. Users can get subscription name by querying Containers table directly
- SkipToken param was added. Users can pass this param to get the next page of resources
- Max number of resources per page was decreased from 5000 to 1000 to align the server configurations. Users can get the next pages by using -SkipToken param

Issue https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/893

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.8.0'
```
Comment (BethanyZhou) [inline src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1:15]: Please revert the change, we will bump version for these modules whose upcoming release section in changelog has some contents.

File: src/ResourceGraph/ResourceGraph/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 0.9.0
+* Added support for the new api version with the ability to query with management group scopes using -ManagementGroup param.
+* Deprecated parameter -Include.
+* Introduced -SkipToken param and aligned max resources returned per page with server value.
+
```
Comment (BethanyZhou) [inline src/ResourceGraph/ResourceGraph/ChangeLog.md:26]: Please move change log msgs to the top of ##upcoming release. Refer to: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md#change-log

File: src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -25,5 +25,5 @@
 [assembly: CLSCompliant(false)]
 [assembly: Guid("cdfd4e1b-bb5f-439b-8843-e6127cd9b8ad")]
 
-[assembly: AssemblyVersion("0.8.0")]
-[assembly: AssemblyFileVersion("0.8.0")]
+[assembly: AssemblyVersion("0.9.0")]
```
Comment (BethanyZhou) [inline src/ResourceGraph/ResourceGraph/Properties/AssemblyInfo.cs:28]: Please revert these changes bc we have version controller do such version bump things.

File: src/ResourceGraph/ResourceGraph.Test/ResourceGraph.Test.csproj [hunk 4]
```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
```
Comment (BethanyZhou) [inline src/ResourceGraph/ResourceGraph.Test/ResourceGraph.Test.csproj:14]: Please wait to the SDK general available, localfeed is not allowed for the time being.

File: src/ResourceGraph/ResourceGraph/ResourceGraph.csproj [hunk 5]
```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceGraph" Version="2.1.0" />
```
Comment (BethanyZhou) [inline src/ResourceGraph/ResourceGraph/ResourceGraph.csproj:14]: Please wait to the SDK general available, localfeed is not allowed for the time being.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/ResourceGraph/ResourceGraph.Test/ResourceGraph.Test.csproj (hunk 4 above)
Comment (fenwickt) [reply src/ResourceGraph/ResourceGraph.Test/ResourceGraph.Test.csproj:14]: It's available now :)

**Review by BethanyZhou — APPROVED** [review]
LGTM
